deploy/ansible/roles/legion_core_chart/templates/legion-core-values.yaml.j2

```diff
@@ -79,12 +79,24 @@ grafana:
     tag: "{{ legion_version }}"
 
   ingress:
+{% if dex.enabled == true %}
```

Comment:
The conditional logic should not be inside values file. Ingress templates is a more proper place for that.